drivers/mmc/core/core.c

```diff
@@ -68,9 +68,26 @@ static const unsigned freqs[] = { 400000, 300000, 200000, 100000 };
  * Enabling software CRCs on the data blocks can be a significant (30%)
  * performance cost, and for other reasons may not always be desired.
  * So we allow it it to be disabled.
+ * 
+ * SysFs interface :
+ *
+ * /sys/module/mmc_core/parameters/crc
+ *
+ * Enable / Disable CRC 
+ * 
+ * echo N > /sys/module/mmc_core/parameters/crc (Disabled) or
+ * echo 0 > /sys/module/mmc_core/parameters/crc (Disabled)
+ *
+ * echo Y > /sys/module/mmc_core/parameters/crc (Enlabled) or
+ * echo 1 > /sys/module/mmc_core/parameters/crc (Enabled)
  */
-bool use_spi_crc = 1;
-module_param(use_spi_crc, bool, 0644);
+
+int use_spi_crc = 1;
+EXPORT_SYMBOL(use_spi_crc);
+module_param_named(crc, use_spi_crc, int, 0644);
```

Comment:
There is no need to change it to int. bool is int internally. For example, similar patch from you previous PR uses bool https://github.com/CruelKernel/samsung-exynos9820/commit/6f6e1511cbcdbc2ac5a4731a4a1348170823a93b

Reply:
Yeah, I had it saved and it commited. But anyway you need the export symbol thingy

Reply:
No need to use EXPORT_SYMBOL here. It's for external kernel modules only if they want to reference this variable. I tested crc in MTweaks, it correctly detect this setting and allows to switch it.

Reply:
In hktweaks it did not see it for me. It did after this though.

Reply:
Hmm, need to check. MTWeaks support both of this parameters https://github.com/morogoku/MTweaks-KernelAdiutorMOD/blob/d639ed3a35c335d402bafb4463aa33a073301b8f/app/src/main/java/com/moro/mtweaks/utils/kernel/misc/Misc.java#L66